skype/skype-ps/skype/Set-CsMediaConfiguration.md

```diff
@@ -256,64 +235,45 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -EnableG722StereoCodec
-When set to True (the default value) allows the use of the G.722 wideband speech codec.
-G.722 is a standard voice codec commonly used with Voice over IP applications; with a sampling rate of 16 KHz G.722 provides higher voice quality and clarity than many other commonly-used speech codecs.
+### -Instance
+An instance of the Microsoft.Rtc.Management.WritableConfig.Settings.Media.MediaSettings object.
+You can retrieve this object by calling the Get-CsMediaConfiguration cmdlet with a specific Identity.
+You can then assign new values to the properties of that object, and then save those changes by passing the object to the Set-CsMediaConfiguration cmdlet.
 
 ```yaml
-Type: Boolean
-Parameter Sets: (All)
-Aliases: 
-Applicable: Lync Server 2013
+Type: PSObject
+Parameter Sets: Instance
+Aliases:
+Applicable: Lync Server 2010, Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019
 
 Required: False
 Position: Named
 Default value: None
-Accept pipeline input: False
+Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
-### -EnableH264Codec
```

Comment:
I still see this parameter in the code branch I have access to.  

@kenwith, can you ask the Server 2019 team if this is being removed in 2019?  Even so, it seem we should keep the parameter in the docs for older version.

Reply:
Yes, will start an email thread with the PM and include you both.